embulk-output-mysql/src/test/java/org/embulk/output/mysql/AfterLoadTest.java

```diff
@@ -32,7 +32,8 @@
 
 public class AfterLoadTest
 {
-    private static final String BASIC_RESOURCE_PATH = "org/embulk/output/mysql/test/expect/after_load/";
+    private static final String BASIC_RESOURCE_NAME = "org/embulk/output/mysql/test/expect/after_load/";
+    private static final String BASIC_RESOURCE_PATH = "/" + BASIC_RESOURCE_NAME;
```

Comment:
It seems hard to tell the difference between `NAME` and `PATH`. How about renaming them to `..._RELATIVE_PATH` and `..._ABSOLUTE_PATH`?

Reply:
Thanks fixed in [ae2ec04](https://github.com/embulk/embulk-output-jdbc/pull/317/commits/ae2ec04a4f68ef20c35181f0642ca722846e4b8d)

Reply:
`TestingEmbulk.loadYamlResource` is calling `EmbulkTests.readResource`, and `EmbulkTests.readResource` is calling `Resources.getResource` .

Is it not possible to pass the same argument (`BASIC_RESOURCE_ABSOLUTE_PATH` or `BASIC_RESOURCE_RELATIVE_PATH`) to all three methods?